net.technicpack.TechnicLauncher.yml

```diff
@@ -0,0 +1,60 @@
+app-id: net.technicpack.TechnicLauncher
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.openjdk8
+command: run.sh
+build-options:
+    env:
+        PATH: /app/jre/bin:/usr/bin
+finish-args:
+    - --env=PATH=/app/jre/bin:/usr/bin:/app/bin
+    - --socket=x11
+    - --share=ipc
+    - --device=all
+    - --socket=pulseaudio
+    - --share=network
+    - --persist=.technic
+modules:
+    - name: openjdk
+      buildsystem: simple
+      build-commands:
+          - /usr/lib/sdk/openjdk8/install.sh
+
+    - name: techniclauncher
+      buildsystem: simple
+      build-commands:
+          - install -D TechnicLauncher.jar /app/bin/TechnicLauncher.jar
+      sources:
+          - type: file
+            path: TechnicLauncher.jar
+            url: https://launcher.technicpack.net/launcher4/709/TechnicLauncher.jar
```

Comment:
You should use https://github.com/flathub/flatpak-external-data-checker to detect updates.

Reply:
Would be a rotating-url

I do not understand regex tho